.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: Check stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          # disable issues
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          stale-pr-message: |
+            This pull request is stale because it has been open for 30 days with no activity.
+            It will be closed in 7 days unless the `stale` label is removed.
+          stale-pr-label: stale
+          days-before-pr-stale: 30
+          days-before-pr-close: 7
+          exempt-pr-labels: 'roadmap,must have,must have eventually,should have,nice to have,external contribution'
```

Comment:
While for issues, it may make sense to exempt everything that has a "desirability" label, I'd actually also be more aggressive for PRs here. If a roadmap PR is open for more than a few weeks without activity, there is something wrong with it and we should still close it (it can always be recreated).

So I'd actually *only* exempt ``external contribution`` here.